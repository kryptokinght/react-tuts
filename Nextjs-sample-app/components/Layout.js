//import Head from 'next/head';
//import Navbar from './Navbar';

const Head = require('next/head');
const Navbar = require('./navbar');


const Layout = (props) => {
    return (
        <div>
            <Head>
                <title>Sample nextjs App</title>
                <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossOrigin="anonymous" />
            </Head>
            <Navbar />
            <div className="container">
                {props.children}
            </div>
        </div>
    );
}

export default Layout;