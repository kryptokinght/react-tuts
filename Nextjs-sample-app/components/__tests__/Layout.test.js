import React from 'react';
import Layout from '../Layout';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json'; //removes unnecessary JSON from snapshot

describe('<Layout />', () => {
    it('should be defined', () => {
        expect(Layout).toBeDefined();
    });
    it('should render', () => {
        const wrapper = shallow(<Layout />);
        expect(shallowToJson(wrapper))
        .toMatchSnapshot();
    });
});