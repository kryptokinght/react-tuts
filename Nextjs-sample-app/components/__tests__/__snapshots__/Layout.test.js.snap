// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> should render 1`] = `
<div>
  <SideEffect(Head)>
    <title>
      Sample nextjs App
    </title>
    <link
      crossOrigin="anonymous"
      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css"
      integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB"
      rel="stylesheet"
    />
  </SideEffect(Head)>
  <Navbar />
  <div
    className="container"
  />
</div>
`;
